docs/source/TROUBLESHOOTING.rst

```diff
@@ -0,0 +1,90 @@
+.. _TROUBLESHOOTING:
```

Comment:
The formatting is inconsistent for the issues.  Either update all of the old ones with **Explanation** and **Solution** or update the docker ones to look like the rest.

Reply:
Updated docker issues to match structure of the rest.